webapp/channels/src/components/add_users_to_team_modal/add_users_to_team_modal.tsx

```diff
@@ -259,7 +259,7 @@ export class AddUsersToTeamModal extends React.PureComponent<Props, State> {
                         buttonSubmitLoadingText={buttonSubmitLoadingText}
                         saving={this.state.saving}
                         loading={this.state.loading}
-                        placeholderText={localizeMessage({id: 'multiselect.placeholder', defaultMessage: 'Search and add members'})}
+                        placeholderText={defineMessage({id: 'multiselect.placeholder', defaultMessage: 'Search and add members'})}
```

Comment:
whats stopping us from passing the translated text directly instead of padding defineMessage

Reply:
Nothing really. I'm undecided on which method I like better since they both have pros and cons. I'm leaning towards passing `MessageDescriptors` since it lets the child component decide when and how to render the message, but it does make passing values into the string much harder.

In general, I ended up letting these components support whatever made the diff more straightforward. I originally tried to remove `injectIntl` from a bunch of these components in favour of `defineMessage`, but I undid all of that because it made the PR even bigger. In this case, it seems like I forgot to use the injected intl object here, but I don't think that's much of a concern